Eatery/Controllers/Eateries/EateriesViewController.swift

```diff
@@ -523,6 +523,14 @@ class EateriesViewController: UIViewController {
         return (favorites: favorites, open: open, closed: closed)
     }
 
+    func favoritesToggled(_ state: Bool) {
```

Comment:
`toggleFavorites` to keep consistent with line 529. If this only _reacts_ to a change in state, then it should be `didToggleFavorites`. Seems like this function is changing the state though, so do the latter.